### PR TITLE
Remove references to the rejectedCommand and to detailedState

### PR DESCRIPTION
**Pull request by menanteau**

### CODE REVIEW

File: python/salpytools/salpylib.py [hunk 1]
```diff
@@ -107,25 +106,24 @@ def __init__(self, Device='atHeaderService', default_state='OFFLINE',
             self.load_state_enumeration()
         except Exception:
             LOGGER.warning("Cannot load state enumeration -- will use library instead")
-            self.detailedState_enum = states.state_enumeration
+            self.State_enum = states.state_enumeration
```
Comment (mareuter) [inline python/salpytools/salpylib.py]: I think this should be self.summaryState_enum to be explicit. Ditto for all the lines below that use this.

File: python/salpytools/salpylib.py [hunk 2]
```diff
@@ -107,25 +106,24 @@ def __init__(self, Device='atHeaderService', default_state='OFFLINE',
...
 
     def load_state_enumeration(self):
         """
-        Load up into dictionaries the SummaryState and DetailedState
+        Load up into dictionaries the State
         enumeration by getting the attributes of the shape
-        'atHeaderService_shared_SummaryState_DisabledState' and
-        'atHeaderService_shared_DetailedState_DisabledState' for example
+        'SALPY_ATCamera.SAL__STATE_DISABLED'
```
Comment (mareuter) [inline python/salpytools/salpylib.py:115]: Shouldn't that be ATHeaderService or is it just a generic holder to point at the state constants?

File: python/salpytools/salpylib.py [hunk 3]
```diff
@@ -107,25 +106,24 @@ def __init__(self, Device='atHeaderService', default_state='OFFLINE',
             self.load_state_enumeration()
         except Exception:
             LOGGER.warning("Cannot load state enumeration -- will use library instead")
-            self.detailedState_enum = states.state_enumeration
+            self.State_enum = states.state_enumeration
 
     def load_state_enumeration(self):
         """
-        Load up into dictionaries the SummaryState and DetailedState
+        Load up into dictionaries the State
```
Comment (mareuter) [inline python/salpytools/salpylib.py]: Leave it as SummaryState instead of just State.

**Review by mareuter — CHANGES_REQUESTED** [review]

File: python/salpytools/salpylib.py (hunk 1 above)
Comment (menanteau) [reply python/salpytools/salpylib.py]: Fixed

File: python/salpytools/salpylib.py (hunk 2 above)
Comment (menanteau) [reply python/salpytools/salpylib.py:115]: Just a generic holder

File: python/salpytools/salpylib.py (hunk 3 above)
Comment (menanteau) [reply python/salpytools/salpylib.py]: Fixed

**Pull request merged**